backend/src/main/java/com/digginroom/digginroom/controller/RoomController.java

```diff
@@ -25,6 +26,12 @@ public ResponseEntity<RoomResponse> showRecommendedRoom(@Auth final Long memberI
         return ResponseEntity.ok().body(roomService.recommend(memberId));
     }
 
+    @GetMapping("/scrap")
+    public ResponseEntity<RoomsResponse> findScrapRooms(@Auth final Long memberId) {
```

Comment:
```suggestion
    @GetMapping("/scrap")
    public ResponseEntity<RoomsResponse> findScrappedRooms(@Auth final Long memberId) {
```
컨트롤러니까
버튼 눌러서 커밋 가시죠 ㅎㅎ